docs/essentials/widget_language.md

```diff
@@ -484,12 +483,12 @@ Animate {
 }
 ```
 
-We have already introduced `id` before. Here, we use it to name the object `greet_new`. `Animate` has three public fields `transition`, `prop` and `from`. These three fields described how an animation runs. 
+We have already introduced the `id` property before. Here, we use it to assign the id `greet_new`. `Animate` has three public fields `transition`, `prop` and `from`. These three fields define how an animation runs. 
 
-- `transition` field accepts a type that implemented `Roc` (Rate of change) trait, Ribir provides `Transition` as the standard implementation. We use `Transition` to describe how property animates smoothly from another value to the current value over time. 
+- The `transition` field accepts a type that implements the `Roc` (Rate of change) trait. Ribir provides `Transition` as the standard implementation. We use `Transition` to describe how a property animates smoothly from some previous value to the new value over time. 
 > **Tips**
 >
-> `Transition` also supports declaring individual, `Animate` can use it across its `id`.
+> `Transition` also supports **declaring individual**(I don't understand this part). `Animate` can access it via its `id`.
```

Comment:
It should be:
`Transition` also can be declared individually. For example, `Animate` can access it via its `id`.